colossalai/zero/init_ctx/init_context.py

```diff
@@ -4,12 +4,12 @@
 import torch
 from colossalai.context.parallel_mode import ParallelMode
 from colossalai.core import global_context as gpc
+from colossalai.logging import disable_existing_loggers, get_dist_logger
```

Comment:
`disable_existing_loggers` is not used in this file and can be removed.

Reply:
DONE